pkg/crc/constants/darwin/constants.go

```diff
@@ -0,0 +1,24 @@
+//+build darwin build
```

Comment:
@cfergeau I got your intention about using the `build` tag here but I think now the directory structure is become bit more confusing then it was before, also since this `build` tag is for all the 3 platform why we need then platform specific tag also?

Directory structure which we have now making me confuse around why there is `constants_darwin.go` and then `darwin/constants.go` :(  cc @gbraad 
```
$ tree pkg/crc/constants/
pkg/crc/constants/
├── common
│   └── constants.go
├── constants_darwin.go
├── constants.go
├── constants_linux.go
├── constants_windows.go
├── darwin
│   └── constants.go
├── linux
│   └── constants.go
└── windows
    └── constants.go
```

Reply:
the directories darwin/constants.go are likely unnecessary when tagged correct with +embedder on the constants_darwin.go